provisioning/device/test/_polling_state_machine_test.js

```diff
@@ -166,7 +182,7 @@ describe('polling state machine', function () {
         });
       });
 
-      /* Tests_SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_18_021: [ If `PollingTransport.queryOperationStatus` succeeds with status==Assigning, `register` shall emit an 'operationStatus' event and begin polling for operation status requests. ] */
```

Comment:
>emit an 'operationStatus' event and  [](start = 164, length = 36)

and this too and i don't know why we're changing the behavior

Reply:
See previous comments.

---
In reply to: [282194000](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282194000) [](ancestors = 282194000)